app/backend/aquifers/urls.py

```diff
@@ -18,102 +18,103 @@
 from aquifers import views
 
 CACHE_TTL = 60*15
+api_version_url_re = '(?P<version>(v1|v2))'
 
 urlpatterns = [
-    url(r'^api/v1/aquifers$',
+    url(rf'^api/' + api_version_url_re + r'/aquifers$',
```

Comment:
Minor inconsistency with the other lines of code

Reply:
Good catch - this syntax though is changing in the next commit after a discussion with Steve.